### PR TITLE
fix for webpack 2

### PR DESCRIPTION
**Pull request by freewayspb**
fix for use loaders with webpack2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ module.exports = {
   ...
   module: {
     loaders: [
```
Comment (graingert) [inline README.md]: this should be `rules` now

File: README.md [hunk 2]
```diff
@@ -78,8 +78,8 @@ module.exports = {
...
-      { test: /\.scss$/i, loader: extractCSS.extract(['css','sass']) },
-      { test: /\.less$/i, loader: extractLESS.extract(['css','less']) },
+      { test: /\.scss$/i, loader: extractCSS.extract(['css-loader','sass-loader']) },
```
Comment (tleunen) [inline README.md]: Should `loader` be `use` now as well?

Comment (Kovensky) [inline README.md]: Related: https://github.com/webpack/extract-text-webpack-plugin/issues/265

The problem is that, if you use `use:`, you get a crash.

**Comment by SpaceK33z** [comment]
Closing in favor of #349.

**Pull request closed without merging**